modules/internal/ChapelRange.chpl

```diff
@@ -398,7 +398,7 @@ module ChapelRange {
           " compile with -snewRangeLiteralType.");
     return oldRule;
   }
-  private proc isValidRangeIdxType(type t) param {
+  proc isValidRangeIdxType(type t) param {
```

Comment:
Since this is auto-used (? fact check me on that), I think you may need to rename this to have an underscore of `chpl_` prefix. Otherwise, you may have just non-backwards-compatibly taken away a user symbol (`isValidRangeIdxType`).